content/kubermatic/master/tutorials_howtos/external_clusters/eks/_index.en.md

```diff
@@ -5,7 +5,54 @@ weight = 7
 
 +++
 
+## Add EKS Cluster
 
+You can add an existing Kubernetes cluster and then manage it using KKP.
+From the `Clusters` page, click `External Clusters`. Click the `Add External Cluster` button and pick `Elastic Kubernetes Engine` provider.
 
+![Add External Cluster](/img/kubermatic/master/tutorials/external_clusters/add_external_cluster.png "Add External Cluster")
 
+Select preset with valid credentials or enter EKS `Access Key ID`, `Secret Access Key` , and `Region` to connect to the provider.
+
+![EKS credentials](/img/kubermatic/master/tutorials/external_clusters/eks_credentials.png "EKS credentials")
+
+You should see the list of all available clusters in the region specified. Select the one and click the `Import Cluster` button. Clusters can be imported only once in a single project. The same cluster can be imported for the other projects.
+
+![Select EKS cluster](/img/kubermatic/master/tutorials/external_clusters/select_eks_cluster.png "Select EKS cluster")
+
+## Cluster Details Page
+
+After the cluster is added, you can view the cluster details.
+The KKP controller retrieves the cluster kubeconfig to display all necessary information. A healthy cluster shows `Running` state.
+Otherwise, the cluster can be in the `Error` state. Move the mouse cursor over the state indicator to get more details.
+
+![EKS cluster](/img/kubermatic/master/tutorials/external_clusters/eks.png "EKS cluster")
+
+You can also click on `Machine Deployments` to get the details:
+
+![EKS Machine Deployment](/img/kubermatic/master/tutorials/external_clusters/eks_machine_deployments.png "EKS Machine Deployment")
+
+## Update Cluster
+
+### Upgrade Version
+
+To upgrade, click on the little dropdown arrow beside the `Control Plane Version` on the cluster’s page and specify the version. For more details about AWS available Kubernetes versions
+[AWS Kubernetes Version](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html "AWS Kubernetes Version")
```

Comment:
This page lists "Amazon EKS Kubernetes versions". I'd switch to the EKS naming there instead of using AWS.